bucket/resharper-clt.json

```
@@ -1,6 +1,8 @@
 {
     "version": "2019.2",
     "homepage": "https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt",
+    "description": "A free set of standalone command line tools that enable you to run ReSharper inspections outside of Visual Studio and find duplicate code.",
```

Resharper do not offer only duplicate finder

```suggestion
    "description": "A standalone command line tools to run ReSharper inspections outside of Visual Studio.",
```